src/containers/Experiment/ProcessingInformationCell.js

```diff
@@ -0,0 +1,394 @@
+
+
+import React from 'react';
+import Button from '../../components/Button';
+import ModalManager from '../../components/Modal/ModalManager';
+
+import FileIcon from './file.svg';
+import ProcessIcon from './process.svg';
+
+
+// Pipeline names ref: https://github.com/AlexsLemonade/refinebio-frontend/issues/22#issuecomment-394010812
+const PIPELINES = {
```

Comment:
With the introduction of `Processor` introduced in each `ComputationalResult` record, this variable may have to be updated. See:
https://github.com/AlexsLemonade/refinebio/blob/2e48ea9b16d2b5a3b0ffa074d28052800a746571/common/data_refinery_common/models/models.py#L410
We probably should figure out a better way to synchronize these constants than hardcoded strings. Can you file a new issue to keep track of it? Thanks.

Reply:
Sure, I added the issue here https://github.com/AlexsLemonade/refinebio-frontend/issues/249 for further discussion.